packages/firebase_core/firebase_core/pubspec.yaml

```diff
@@ -18,7 +18,7 @@ flutter:
         default_package: firebase_core_web
 
 dependencies:
-  firebase_core_platform_interface: ^1.0.0
+  firebase_core_platform_interface: ^1.0.4
```

Comment:
Other than to ensure deps to be re-downloaded, this shouldn't be needed, right?

Reply:
1.0.4 is needed at minimum to make sure the tests pass

Reply:
Ahhh because of the new internal implementation of the `FirebaseCorePlatform`, makes sense!